### PR TITLE
Add a FAQ about the ocm-backplane binary

### PR DESCRIPTION
**Pull request by feichashao**
### What type of PR is this?

documentation

### Which Jira/Github issue(s) does this PR fix?

https://issues.redhat.com/browse/OSD-19782

### Pre-checks (if applicable)

- [ ] Ran unit tests locally
- [ ] Validated the changes in a cluster
- [ ] Included documentation changes with PR

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Frequently Asked Questions
+
+## Why the binary name is ocm-backplane ?
```
Comment (supreeth7) [inline docs/FAQ.md:3]:
```suggestion
## Why is the binary named ocm-backplane?
```

File: docs/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+With the name `ocm-*`, the binary can act as an [OCM plugin](https://github.com/openshift-online/ocm-cli/blob/119e9d2e8af0ddf5db4b0888b5ceadb300768885/README.md#extend-ocm-with-plugins).
+
+The logic behind OCM plugin:
+- The user runs `ocm backplane`.
+- `ocm` doesn't have the `backplane` subcommand.
+- `ocm` lookup the PATH and find the `ocm-backplane` binary.
```
Comment (supreeth7) [inline docs/FAQ.md:9]:
```suggestion
- `ocm` looks up the `$PATH` and finds the `ocm-backplane` executable binary.
```

File: docs/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+- `ocm` calls the `ocm-backplane` binary.
+
+## Does ocm-backplane depend on a specific OCM binary version?
+No. The `ocm-backplane` is an independent binary. It is fully functional without the `ocm` binary. The interaction with OCM api will be handled by ocm-sdk, which is defined in the go.mod of ocm-backplane.
```
Comment (supreeth7) [inline docs/FAQ.md:13]:
```suggestion
`ocm-backplane` is not a standalone binary. It is functional even without the `ocm` binaries. The interaction with OCM API will be handled by OCM SDK, which is defined as a dependency of the project.
```

**Review by supreeth7 — CHANGES_REQUESTED** [review]
Nice idea, looking at the adoption of Backplane outside SRE, we really need a dedicated FAQ in the repository.

File: docs/FAQ.md (hunk 3 above)
Comment (feichashao) [reply docs/FAQ.md:13]: Thanks @supreeth7 !   Did you mean `not a standalone` or `a standalone`?

Comment (supreeth7) [reply docs/FAQ.md:13]: Right, my bad, ~not~

**Comment by feichashao** [comment]
@supreeth7  thanks for reviewing and fixing the grammar!  I committed changes and squashed.

**Review by supreeth7 — APPROVED** [review]
/lgtm

**Pull request merged**